ern-api-impl-gen/src/ApiImplRegen.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import ApiImplGen from './generators/ApiImplGen'
+import { Dependency } from 'ern-util'
+
+export default async function regenerateApiImpl
+  ({
+     api,
+     paths,
+     reactNativeVersion,
+     platforms
+   }: {
+    api: Dependency,
+    paths: Object,
+    reactNativeVersion: string,
+    platforms: Array<string>
+  }) {
+  try {
+    log.debug(`Entering regenerateApiImpl ${JSON.stringify(arguments[0])}`)
+    await new ApiImplGen().generateApiImplementation(api, paths, reactNativeVersion, platforms, true)
+  } catch (e) {
+    throw new Error(e)
```

Comment:
please just `throw(e)` to `rethrow` the error.
actually you dont need a `try/catch` block at all here as you don't do anything specific in it but you just retrhow.